orchestrator/test_runner/src/airdrop_proposal.rs

```diff
@@ -31,7 +31,9 @@ pub async fn airdrop_proposal_test(contact: &Contact, keys: Vec<ValidatorKeys>)
     // we start by disabling inflation, this allows us to check that the correct
     // amount of tokens was withdrawn from the community pool, if we don't do this inflation
     // in the community pool may be larger than our actual airdrop amount spent.
-    disable_inflation(contact, &keys).await;
```

Comment:
Here I refactored the vote_yes_on_proposals() function so that we can submit proposals to the ibc chain, which doesn't have the necessary &[ValidatorKeys] type, and instead has &[CosmosPrivateKey]